cartoframes/data/observatory/provider.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
+
+from cartoframes.data.observatory.repository.provider_repo import get_provider_repo
```

Comment:
just realized according to `our convention` we forgot to use relative imports in this PR.